android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -274,51 +279,73 @@ private void doMenuAction(int menuItemId) {
         FragmentManager fragmentManager = getSupportFragmentManager();
         switch (menuItemId) {
             case R.id.nav_tracks:
+                addShadowToAppBar(true);
```

Comment:
Since for all the nav drawer options you have to set the shadow to true and only for schedule as false, I suggest to remove the common code and call the function to true in the beginning.